server/app.ts

```diff
@@ -40,6 +44,8 @@ export default function createApp(controllers: Controllers, services: Services):
   app.use(routes(controllers))
 
   app.use((req, res, next) => next(createError(404, 'Not found')))
+  // The error handler must be before any other error middleware and after all controllers
+  setUpSentryErrorHandler(app)
   app.use(errorHandler(process.env.NODE_ENV === 'production'))
```

Comment:
Is it worth being more explicit in these two comments that we're talking about the `Sentry error handler` and the `Sentry request handler`, especially given there's also `errorHandler` on the next line in the former case?